namer/namer.cc

```diff
@@ -2020,6 +2020,20 @@ class TreeSymbolizer {
             }
         }
 
+        // Prevent a crash by validating the arguments before we do any symbol lookups or manipulations
+        if (send->hasPosArgs() || send->hasKwArgs() || send->block() != nullptr) {
+            if (send->numPosArgs() > 1) {
+                if (auto e = ctx.beginError(send->loc, core::errors::Namer::InvalidTypeDefinition)) {
```

Comment:
Ideally this error location would point to the bad arguments, not the whole method call, like what you would get for a normal “too many arguments provided to method” error